.github/workflows/CI.yml

```
@@ -0,0 +1,25 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        node-version: [ 20 ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Setup project
+        uses: bpmn-io/actions/setup@latest
+      - name: Lint
+        run: npm run lint
```

On CI, similar to local development, the script to execute is typically `npm run all`. If CI and local `all` script differs, call the CI one `ci`.

```suggestion
        run: npm run all
```